dashboard/app/models/script.rb

```diff
@@ -123,7 +123,9 @@ def generate_plc_objects
     exclude_csf_column_in_legend
     teacher_resources
     stage_extras_available
+    has_verified_resources
   )
+  # TODO: validate has_verified_resources in .script
```

Comment:
TODO meant to be left here? Is there just not a test file yet?

Reply:
meant to manually verify that setting actually gets persisted. it does, and i just forgot to remove TODO